astronomy/butcher_mercury_orbiter_1119.hpp

```diff
@@ -0,0 +1,45 @@
+﻿#pragma once
```

Comment:
Just call this file `memory_orbiter.hpp`.  No point in recording who provided it or what bug it was associated with.  (You can have that in the comments, of course.)